### ISSUE TITLE
Events header should appear first on xs display

### ISSUE THREAD

**Issue by neilerdwien** [neilerdwien/homepage-2017#29]
Perhaps the flex order can be used for this?

Otherwise, a duplicate header with only one shown at a time would work.

Should the whole list appear first?

**Comment by jcorkill** [comment]
I agree. The Events header needs to be there. I don't think it would bother me if the list appeared first. Then you don't have to duplicate and it would match News.
